_includes/help-build-fail.md

```diff
@@ -6,4 +6,29 @@ The files we are using are very picky. If you made any mistakes, you will see a
 
 When you commit your changes, Travis CI will re-check your file.
 
-Keep doing this until you see a message that the build has passed. If you need help, don't forget you can find us in chat or @ mention githubteacher in your pull request.
+Keep doing this until you see a message that the build has passed. If you need help, don't forget you can find us in chat or @ mention `@githubteacher` in your pull request.
+
+### Help With Specific Issues
+
+The build can fail because of many reasons. By looking at the details of the **Travis CI** log it might help you to troubleshoot your code.
+
+1. Open the _Pull request_  
```

Comment:
😍 This is such an awesome addition. Reading it now, it feels like it fits and belongs so naturally in the current content!